### PR TITLE
Mat 2017 02 26 nodeCreation

### PR DESCRIPTION
**Pull request by mjfortier**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -8,6 +8,10 @@ def static(path):
     return bottle.static_file(path, root='static/')
 
 
+@bottle.route('/')
+def index():
+    return 'serving is running successfully'
```
Comment (john-swu) [inline app/main.py]: good idea.

File: app/test.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+MAPSIZEX = 20       #Global variable for X size of map (up-down)
```
Comment (john-swu) [inline app/test.py]: I would not have any tests in the base snake. That's something we should leave up to individuals as they fork it.

File: app/test.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+MAPSIZEY = 20       #Global variable for Y size of map (left-right)
+
+class Node:         #Class for spaces on the board, refered to as Nodes
```
Comment (john-swu) [inline app/test.py]: this is the pythonic way in python3, in python 2 it's good form to extend Object, i.e. `class Node(Object):`

**Comment by codeallthethingz** [comment]
Closing this as we're repurposing for the new api

**Pull request closed without merging**